pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -142,47 +148,65 @@ func (h *NetworkUtilsHandler) UseIptables() bool {
 
 	return true
 }
-func (h *NetworkUtilsHandler) IptablesNewChain(table, chain string) error {
-	iptablesObject, err := iptables.New()
+
+func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
+	_, err := exec.Command("sysctl", "net.ipv6.conf.all.forwarding=1").CombinedOutput()
+	return err
+}
+
+func (h *NetworkUtilsHandler) IsIpv6Enabled(link netlink.Link) bool {
```

Comment:
this function isn't using the `link` argument at all. It could be a bit deceiving

Reply:
You're right! Done.